build.gradle

```diff
@@ -47,8 +48,13 @@ group = 'com.android.volley'
 version = '1.2.0-SNAPSHOT'
 
 android {
-    compileSdkVersion 25
-    buildToolsVersion = '25.0.2'
+    compileSdkVersion 28
+    buildToolsVersion = '28.0.3'
+
+    defaultConfig {
+        minSdkVersion 8
```

Comment:
is it necessary to specify this here since it's already in the AndroidManifest after your latest commit?

it's minor, but better if it's only specified in one place, or else we might update one in the future and be confused depending on which one takes precedence.

Reply:
Gotcha.  I switched it to a build variable per https://developer.android.com/studio/build/manifest-build-variables .  I'm not sure how I can test the alternate build system to make sure it works there, any thoughts?

Reply:
Is it not working if you just leave the minSdkVersion out of build.gradle entirely and have it hard-coded in the manifest, like how we had it before? Or does the new gradle version make this a required field for some reason?

Reply:
New gradle plugin requires it, else it assumes 1

Reply:
Ah okay, that's what I was missing.

In that case let's have it explicitly set as 8 in both places. Just please add a comment to each along the lines of:

```
defaultConfig {
  // Keep in sync with src/main/AndroidManifest.xml
  minSdkVersion 8
}
```

and

```
<!-- Keep in sync with build.gradle -->
<uses-sdk minSdkVersion="8" />
```